Dockerfile

```
@@ -0,0 +1,24 @@
+FROM tomcat:8-jdk8-openjdk
```

Do we need the JDK installed in the image?
Also, should we switch to `temurin` here as well?

```suggestion
FROM tomcat:8-jre8-temurin
```

The difference in size between those 2 base images is 300MB. It would however mean using `curl -sLO` instead of `wget -q` below, and installing `unzip`.